test/wasm/test_openmp.cpp

```diff
@@ -46,4 +46,10 @@ namespace tests {
         message::Message msg = util::messageFactory("omp", "stack_debug");
         execFunction(msg);
     }
+
+    TEST_CASE("Test simple reduction function", "[wasm]") {
+        cleanSystem();
+        message::Message msg = util::messageFactory("omp", "simple_reduce");
+        execFunction(msg);
+    }
```

Comment:
Would be nice to add some more of the `omp` functions as tests to get a broader coverage.

Reply:
This is done, there is a lot of constructs tested now. I adapted the averaging and the integral code I found online to now test the return values